src/libpriv/rpmostree-postprocess.c

```diff
@@ -964,13 +964,23 @@ create_rootfs_from_yumroot_content (GFile         *targetroot,
 
   g_print ("Adding tmpfiles-ostree-integration.conf\n");
   {
-    gs_unref_object GFile *src_pkglibdir = g_file_new_for_path (PKGLIBDIR);
-    gs_unref_object GFile *src_tmpfilesd =
-      g_file_get_child (src_pkglibdir, "tmpfiles-ostree-integration.conf");
+    /* This is useful if we're running in an uninstalled configuration, e.g.
+     * during tests. */
+    const char *conf = g_getenv("RPMOSTREE_TMPFILES_CONF");
```

Comment:
I'd call this `RPMOSTREE_UNINSTALLED_PKGLIBDIR` - if we can assume the directory we point it to resembles the installed location.  We might end up having to create an "installed" directory as part of `make` that symlinks to the source dirs or something, but right now this is the only file that's there.